Makefile

```diff
@@ -20,6 +20,9 @@ distclean: clean
 
 -include config.mk
 
+setup.ml: _oasis
+	@oasis setup
+
```

Comment:
Is that standard? I quite like to do the same thing for my projects... I get tired of that autogen crap.